example-crates/wallet_electrum/src/main.rs

```diff
@@ -89,7 +93,7 @@ fn main() -> Result<(), anyhow::Error> {
     assert!(finalized);
 
     let tx = psbt.extract_tx()?;
-    client.transaction_broadcast(&tx)?;
+    client.inner.transaction_broadcast(&tx)?;
```

Comment:
⛏️ Can we also expose `transaction_broadcast()` as`client.transaction_broadcast(..)`? Most users will only be doing `sync()`, `full_scan()` and `transaction_broadcast()`.